### DISCUSSION TITLE
Example documentation location

### DISCUSSION THREAD

**Discussion by dooglewoogle** [pystiche/pystiche#589] [Q&A] [answered]
Hi I'm trying to write some additions to the Model Optimization example documentation, specifically around obtaining and loading a dataset. 
Is this documentaton generated from the .py file, or is there a .rst I've missed?
Thanks

**Comment by pmeier** [comment, marked as answer]
Hey and welcome @dooglewoogle! The usage examples are generated by [`sphinx-gallery`](https://github.com/sphinx-gallery/sphinx-gallery). You can find its config here:

https://github.com/pystiche/pystiche/blob/c6069f991f4cb78584f5c82dc8545cc97e96205d/docs/source/conf.py#L280-L283

It will build all files in `example_dirs` that match `filename_pattern`. You are looking for 

https://github.com/pystiche/pystiche/blob/c6069f991f4cb78584f5c82dc8545cc97e96205d/examples/advanced/example_model_optimization.py#L303

I'm happy you want to contribute! You can follow the [contribution guide](https://docs.pystiche.org/en/latest/getting_started/contributing.html#documentation) to see how you can build the documentation. If you only want to change text or code that does not affect the output, I suggest you run with `PYSTICHE_PLOT_GALLERY=0` to avoid building the examples, because that will take some time.
